examples/index.md

```diff
@@ -162,3 +162,209 @@ Content-Type: application/vnd.api+json
+HTTP/1.1 422 Unprocessable Entity
+Content-Type: application/vnd.api+json
+
+{
+  "errors": [
+    {
+      "title": "Adapter Error",
+      "code": "ds.errors.invalid-error-expects-json-api-format",
+      "detail": "The adapter rejected the commit because it was invalid"
+    }
+  ]
+}
+```
+
+Response with status:
+
+```http
+HTTP/1.1 422 Unprocessable Entity
+Content-Type: application/vnd.api+json
+
+{
+  "errors": [
+    {
+      "status": "403",
+      "source": { "pointer": "data/attributes/secret-powers" },
+      "detail": "Access to secret powers not authorized on Sundays."
+    },
+    {
+      "status": "404",
+      "source": { "pointer": "data/attributes/vorpal-blade" },
+      "detail": "'Vorpal Blade' is not an attribute of mimsy borogroves."
+    },
+    {
+      "status": "422",
+      "source": { "pointer": "data/attributes/volume" },
+      "detail": "Volume does not, in fact, go to 11."
+    },
+    {
+      "status": "500",
+      "title": "The backend responded with an error",
+      "detail": "Reputation service not responding after three requests."
+    },
+    {
+      "status": "400",
+      "detail": "JSON parse error - Expecting property name enclosed in double quotes: line 1 column 2 (char 1)"
+    }
+  ]
+}
+```
+
+The only uniqueness constraint on error objects in a response is the `id` field.
+Multiple errors on the same attribute may be described by an error object
+for each error.
+
+Multiple errors on `first-name` attribute:
+
+```http
+HTTP/1.1 422 Unprocessable Entity
+Content-Type: application/vnd.api+json
+
+{
+  "errors": [
+    {
+      "title": "Invalid Attribute",
+      "source": { "pointer": "data/attributes/first-name" },
+      "detail": "First name must contain at least three characters."
+    },
+    {
+      "title": "Invalid Attribute",
+      "source": { "pointer": "data/attributes/first-name" },
+      "detail": "First name must contain an emoji."
+    }
+  ]
+}
+```
+
+If the API docs specified:
+
+> | Error Id   | Title                                                                             |
+> |------------|-----------------------------------------------------------------------------------|
+> |  11        | "Invalid Attribute"                                                               |
+> |  123       | "too short"                                                                       |
+> |  124       | "emoji missing"                                                                   |
+> |  225       | "password must contain a letter, number, space, neologism, and special character" |
+> |  226       | "passwords do not match"                                                          |
+> |  227       | "password cannot be one of last five passwords"                                   |
+
+Multiple errors on `password` attribute, with error "id":
+
+```http
+HTTP/1.1 422 Unprocessable Entity
+Content-Type: application/vnd.api+json
+
+{
+  "errors": [
+    {
+      "id":   "123",
```

Comment:
Updated. I'll rebase if it looks good. 

So, 'id' could be a uuid generated at error time and logged as part of that request/response. Gotcha.